LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -2128,6 +2128,56 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
 
                             break;
                         }
+                        case Model.TUF_GAMING_B550M_PLUS_WIFI: //NCT6798D
+                        {
+                            v.Add(new Voltage("Vcore", 0));
+                            v.Add(new Voltage("Voltage #2", 1, true));
+                            v.Add(new Voltage("AVCC", 2, 34, 34));
+                            v.Add(new Voltage("+3.3V", 3, 34, 34));
+                            v.Add(new Voltage("Voltage #5", 4, true));
+                            v.Add(new Voltage("Voltage #6", 5, true));
+                            v.Add(new Voltage("Voltage #7", 6, true));
+                            v.Add(new Voltage("3VSB", 7, 34, 34));
+                            v.Add(new Voltage("VBat", 8, 34, 34));
+                            v.Add(new Voltage("VTT", 9));
+                            v.Add(new Voltage("Voltage #11", 10, true));
+                            v.Add(new Voltage("Voltage #12", 11, true));
+                            v.Add(new Voltage("Voltage #13", 12, true));
+                            v.Add(new Voltage("Voltage #14", 13, true));
+                            v.Add(new Voltage("Voltage #15", 14, true));
+                            t.Add(new Temperature("System", 0));
+                            t.Add(new Temperature("CPU", 1));
+                            t.Add(new Temperature("AUX 0", 2));
+                            t.Add(new Temperature("AUX 1", 3));
+                            t.Add(new Temperature("AUX 2", 4));
+                            t.Add(new Temperature("AUX 3", 5));
+                            t.Add(new Temperature("AUX 4", 6));
+                            t.Add(new Temperature("SMBus 0", 7));
+                            t.Add(new Temperature("SMBus 1", 8));
+                            t.Add(new Temperature("PECI 0", 9));
+                            t.Add(new Temperature("PECI 1", 10));
+                            t.Add(new Temperature("PCH Chip CPU Max", 11));
+                            t.Add(new Temperature("PCH Chip", 12));
+                            t.Add(new Temperature("PCH CPU", 13));
+                            t.Add(new Temperature("PCH MCH", 14));
+                            t.Add(new Temperature("Agent 0 Dimm 0", 15));
+                            t.Add(new Temperature("Agent 0 Dimm 1", 16));
+                            t.Add(new Temperature("Agent 1 Dimm 0", 17));
+                            t.Add(new Temperature("Agent 1 Dimm 1", 18));
+                            t.Add(new Temperature("Byte 0", 19));
+                            t.Add(new Temperature("Byte 1", 20));
+                            t.Add(new Temperature("PECI 0 Cal", 21));
+                            t.Add(new Temperature("PECI 1 Cal", 22));
```

Comment:
Cal of Calibrate? Or something else? Might be worth writing this out.

Reply:
That's something which is actually a bit better explained in the docs but still somewhat far from logical.
PECI is the Intel interface used to report CPU temperatures to the mainboard. It should not work on AMD boards and does not on my system (values PECI 0 and PECI 1). So far so good.
But the docs now explain that PECI values are notoriously imprecise, so there is an option to "automatically calibrate" them based on temperature probes (though no explanation what probe is used). For some unknown reasons this value ("PECI 0 Calibration") is the value used by my ASUS board to control the fan speeds. It's showing a value that's correlated to CPU temperature, though usually about 10 K lower. That's actually the value I did all the work for, because I wanted to see the temperature that controls fan speed.
Officially it's called "PECI Agent 0 Calibration".

Reply:
Then I think these sensor names should end with Calibration, it makes more sense than just Cal here and there's room for it in the UI 🙂

Reply:
There is actually something about that in the docs, but it doesn't make a lot of sense, either.
PECI is the Intel interface to report CPU temperatures to the board. They should not work on AMD and they (PECI 0 and PECI 1) are indeed no used on my board.
But now the docs claim that PECI values are notoriously bad, so they can be "calibrated" (what ever that means) by using temperature probes. There is some explanation about how calibration work (it's automagic), but not even which temperature probes are used.
Oddly enough, "PECI 0 Calibration" shows values on my system (which is AMD) and those values are actually used to control fan speeds. The value is roughly correlated to CPU temperature (read out directly), but usually about 10 to 15 K lower.
It's actually the value I did all the work for.

Reply:
Yes, will do.